qiskit/optimization/converters/quadratic_program_converter.py

```diff
@@ -29,7 +28,7 @@ class QuadraticProgramConverter(ABC):
     @abstractmethod
     def convert(self, problem: QuadraticProgram) -> QuadraticProgram:
         """
-        Convert an QuadratciProgram into another form
+        Convert a QuadratciProgram into another form
```

Comment:
```suggestion
        Convert a QuadraticProgram into another form
```
This `QuadratciProgram` still seems to be spelled wrong